test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/Expressions/SelectExpandBinderTest.cs

```diff
@@ -513,11 +513,11 @@ public void ProjectAsWrapper_Element_ProjectedValueContains_SelectedStructuralPr
 
         [Theory]
         [InlineData("Emails", new[] { "E1", "E3", "E2" })]
-        // [InlineData("Emails($orderby=$it)", new[] { "E1", "E2", "E3" })] TODO: enable it when ODL supports $this
```

Comment:
Why were these tests using $it instead of $this?

Reply:
Because they were relying on a bug in ODL which we fixed in this PR https://github.com/OData/odata.net/pull/1906
Once we fixed the bug, we need to fix the WebApi to use `$this`